roles/docker/tasks/main.yml

```diff
@@ -36,6 +36,15 @@
   tags:
     - prebake-for-dev
 
+- stat: path=/var/docker_images
+  register: docker_images
```

Comment:
oh, may be I got it wrong. I thought @unclejack wanted to run this task as part of packer provisioner. If not then we don't need to tag this task at all.